lambdanote/src/Components/NoteList.js

```diff
@@ -3,16 +3,15 @@ import Note from './Note';
 import { Link } from 'react-router-dom';
 import { Container, Row, Button } from 'reactstrap';
 
+
 const NoteList = (props) => {
     return (
          <div>
-            <div className='note-container'>
-            <h2>Your Notes:</h2>
-            <Container fluid>
-            <Row>
-                {props.notes.map(note => <Note key={note.id} note={note} />)}
-            </Row>
-            </Container>
+            <div >
+            <h3 className='notelist-header'>Your Notes:</h3>
+            <div>            
```

Comment:
You should use the flexbox `flex-direction: row` and `flex-wrap: wrap` on this div to get the notes to work right.